.github/workflows/publish-to-pypi.yml

```
@@ -26,10 +31,14 @@ jobs:
         pip install setuptools wheel twine
     - name: Update version in linebot/__about__.py
       run: |
-        VERSION=${{ github.event.release.tag_name }}
+        if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
+          VERSION=${{ github.event.inputs.version }}
+        else
+          VERSION=${{ github.event.release.tag_name }}
+        fi
+
         VERSION=${VERSION#v}
-        sed -i "s/__version__ = '__LINE_BOT_SDK_PYTHON_VERSION__'/__version__ = '$VERSION'/g" linebot/__about__.py
-        cat linebot/__about__.py
+        python tools/update_and_verify_version.py $VERSION
```

Is filename correct...?
```suggestion
        python tools/update_version.py $VERSION
```

---

No 😄 thank you for careful review~